weis/test/run_examples.py

```diff
@@ -3,8 +3,8 @@
 
 all_scripts = [
     "01_aeroelasticse/run_general",
-    #"01_aeroelasticse/run_mass_sweep",
-    #"01_aeroelasticse/run_PC_sweep",
+    "01_aeroelasticse/run_mass_sweep",
+    "01_aeroelasticse/run_PC_sweep",

```

Comment:
I'm fine with taking out these tests and scripts if you are @gbarter

Reply:
Okay, that helps.  Didn't realize you didn't need the capability anymore.